index.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 var normalizeUrl = require('normalize-url');
 
-module.exports = function (a, b) {
+module.exports = function (a, b, options) {
 	if (a === b) {
 		return true;
 	}
-
-	return normalizeUrl(a) === normalizeUrl(b);
+	
+	var opts = Object.assing({}, options);
+	return normalizeUrl(a, opts) === normalizeUrl(b, opts);
```

Comment:
Just pass through the `options` object directly. `normalizeUrl` already [copies](https://github.com/sindresorhus/normalize-url/blob/master/index.js#L36) the object.